.github/delete-azure-resources.sh

```diff
@@ -3,9 +3,10 @@
 # The following environment variables need to be configured before running the script
 # - RESOURCE_GROUP_NAME
 # - STORAGE_ACCOUNT_NAME
-# - APP_CONFIG_NAME
+# - KEY_VAULT_NAME
```

Comment:
Keep `APP_CONFIG_NAME` and add `KEY_VAULT_NAME`?

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/218/commits/1b0f9b4ce9d8e8cfba02237c818ba1bdb4c90574.